tests/responses/hepcrawl/springer/10052_2019_Article_6572.xml.Meta

```diff
@@ -43,6 +43,11 @@
                      <Year>2019</Year>
                      <Month>1</Month>
                   </CoverDate>
+                  <OnlineDate>
```

Comment:
I think in these tests this file won't be parsed, only attached/uploaded to the record.
Tests in hepcrawl repo parse the XML files provided by the publishers and create a json output  (tests/responses/hepcrawl/springer/springer.json) which is used here to test the article_upload workflow. 

So this file shouldn't be changed here, only in the hepcrawl repo, or maybe not changed but added as a new test there.